docs/app/layouts.md

```diff
@@ -0,0 +1,283 @@
+start and end time)_.
+
+<!--
+@TODO Config > Fields
+Link to Datetime Field
+-->
+
+:::
+
+## Map Layout
+
+<video autoplay muted loop controls title="The Map Layout">
+	<source src="" type="video/mp4" />
+</video>
+
+This Layout is ideal for Collections that emphasize geospatial data. It provides a world map, with Items displayed as
+points, lines, and other geometry. The following options are available:
+
+**Layout Options**
+
+- **Basemap** — Choose the map provider used for the Collection.
+- **Geospatial Field** — Select the geospatial field type to display on screen:
+  - **Map JSON Point** — Supports latitude-longitude based, single-point locations.
+  - **Map Geometry** — Supports geometric areas such as lines and polygons.
+- **Display Template** — Choose the Fields displayed when hovering over an Item on the map.
+- **Cluster Nearby Data** — Toggle whether or not nearby Items get clustered into a single pin.
+
+**Subheader**\
+_There is no Subheader on the Map Layout._
+
+**Page Area**
+
+- **Zoom** — Click <span mi icon>add</span> and <span mi icon>remove</span> in the upper left hand corner of the Page
+  Area to zoom in and out.
+- **Zoom to my Location** — Click <span mi icon>my_location</span> to zoom into your current location on the map.
+- **Reframe** — Click the square in the upper left-hand corner to resize and reframe the map area.
+- **Select Item** — Click a single Item to enter its Item Page.
+- **Select Items** — Click and drag to select multiple Items at once, opening the Item Page.
+
+:::tip Configuration Requirements
+
+To use this Layout, the Collection must have a Map Field configured.
+
+<!--
+@TODO Config > Fields
+Link to Map Field
+-->
+
+:::
+
+## Kanban Layout
+
+<video autoplay muted loop controls title="The Kanban Layout">
+	<source src="" type="video/mp4" />
+</video>
+
+This Layout is ideal for Collections that serve as project management tools or to-do lists, where each Item represents a
+task, because it groups Items onto panels according to their status _(e.g. "Not Started", "In Progress", "Under Review",
+"Complete", or any other status that is defined)_. The following controls are available.
+
+**Layout Options**
+
+- **Group By** — Select the Field used to define Item status. See below for details.
+- **Card Title** — Choose a Field to serve as a title on each Item.
+- **Card Text** — Choose a Field to display additional text on each Item.
+
+**Layout Options > Advanced**
+
+- **Card Tags** — Choose a Tag Field to be displayed on the Item.
+- **Card Date** — Choose a Datetime Field to be displayed on each Item.
+- **Card Image** — Choose an Image Field to be displayed on each Item.
+- **Card Image Fit** — Toggle whether the image fit is cropped.
+- **Card User** — Choose the User Created Field to display their avatar in the bottom right corner.
+- **Show Ungrouped** — Toggle display of a Status Panel containing Items with no assigned status.
```

Comment:
Notice that I opted for calling the panels which Items are grouped by "status panels"... Let me know if that's what we want to say or if I should change it.

Reply:
@rijkvanzanten @benhaynes

Reply:
Hmm lets call them "Columns" maybe? To me, they're "cards" on top of "columns" 🤔

Reply:
ok!